scripts/download-browsers.js

```diff
@@ -0,0 +1,97 @@
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+require("dotenv").config();
+
+const util = require("util");
+const path = require("path");
+const {
+  downloadBrowserWithProgressBar,
+} = require("playwright/lib/utils/browserFetcher");
+const { getChromeVersion } = require("./install-pw");
+
+const SUPPORTED_PLATFORMS = ["linux", "mac", "win"];
+const EXECUTABLE_PATHS = {
+  linux: ["chrome-linux", "chrome"],
+  mac: ["chrome-mac", "Chromium.app", "Contents", "MacOS", "Chromium"],
+  win: ["chrome-win", "chrome.exe"],
+};
+
+const CHROMIUM_DOWNLOAD_URLS = {
+  linux: "%s/builds/chromium/%s/chromium-linux.zip",
+  mac: "%s/builds/chromium/%s/chromium-mac.zip",
+  win: "%s/builds/chromium/%s/chromium-win64.zip",
+  // 'mac-arm64': '%s/builds/chromium/%s/chromium-mac-arm64.zip',
+};
+
+async function download(platform, revision, directory) {
+  const executablePath = findExecutablePath(directory, platform);
+  const downloadHost =
+    process.env["PLAYWRIGHT_DOWNLOAD_HOST"] ||
+    "https://playwright.azureedge.net";
+  const downloadURL = util.format(
+    CHROMIUM_DOWNLOAD_URLS[platform],
+    downloadHost,
+    revision
+  );
+  const title = `${platform} v${revision}`;
+  const downloadFileName = `playwright-download-chromium-${platform}-${revision}.zip`;
+  try {
+    // eslint-disable-next-line no-console
+    console.info("Downloading browser ", title);
+    await downloadBrowserWithProgressBar(
+      title,
+      directory,
+      executablePath,
+      downloadURL,
+      downloadFileName
+    );
+  } catch (e) {
+    throw new Error(`Failed to download ${title}, caused by\n${e.stack}`);
+  }
+}
+
+function findExecutablePath(dir, platform) {
+  const tokens = EXECUTABLE_PATHS[platform];
+  return tokens ? path.join(dir, ...tokens) : undefined;
+}
+
+async function downloadAll() {
+  const [, revision] = getChromeVersion().split("-");
+  for (const platform of SUPPORTED_PLATFORMS) {
+    const directory = path.join(
+      process.cwd(),
+      "local-browsers",
+      "_releases",
+      `${getChromeVersion()}-${platform}`
+    );
+    try {
+      await download(platform, revision, directory);
+    } catch (e) {
+      throw Error("Failed to download browser for platform " + platform);
+    }
+  }
+}
+// eslint-disable-next-line no-console
+downloadAll().then(() => console.log("Download succeeded"));
```

Comment:
Move them inside this check if we want to execute only when running as main module and not while importing - https://github.com/elastic/synthetics-recorder/blob/13a146375d4582a04361d057969c3ab17c6e5e42/scripts/install-pw.js#L50